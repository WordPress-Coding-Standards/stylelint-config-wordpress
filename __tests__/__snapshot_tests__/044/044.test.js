'use strict';

const stylelint = require( 'stylelint' ),
	systemTestUtils = require( '../systemTestUtils.js' );

it( '044', () => {
	return stylelint.lint({
		files: [systemTestUtils.caseStylesheetGlob( '044' )],
		configFile: systemTestUtils.caseConfig( '044' ),
	}).then( ({ results }) => {
		expect( systemTestUtils.prepResults( results ) ).toMatchSnapshot();
	});
});
