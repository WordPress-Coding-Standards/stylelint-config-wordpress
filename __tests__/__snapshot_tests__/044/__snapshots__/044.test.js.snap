// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`044 1`] = `
Array [
  Object {
    "deprecations": Array [],
    "errored": true,
    "ignored": undefined,
    "invalidOptionWarnings": Array [],
    "parseErrors": Array [],
    "source": "044/stylesheet.css",
    "warnings": Array [
      Object {
        "column": 43,
        "line": 13,
        "rule": "block-closing-brace-newline-before",
        "severity": "error",
        "text": "Expected newline before \\"}\\" (block-closing-brace-newline-before)",
      },
      Object {
        "column": 26,
        "line": 14,
        "rule": "block-closing-brace-newline-before",
        "severity": "error",
        "text": "Expected newline before \\"}\\" (block-closing-brace-newline-before)",
      },
      Object {
        "column": 66,
        "line": 15,
        "rule": "block-closing-brace-newline-before",
        "severity": "error",
        "text": "Expected newline before \\"}\\" (block-closing-brace-newline-before)",
      },
      Object {
        "column": 56,
        "line": 16,
        "rule": "block-closing-brace-newline-before",
        "severity": "error",
        "text": "Expected newline before \\"}\\" (block-closing-brace-newline-before)",
      },
      Object {
        "column": 64,
        "line": 17,
        "rule": "block-closing-brace-newline-before",
        "severity": "error",
        "text": "Expected newline before \\"}\\" (block-closing-brace-newline-before)",
      },
      Object {
        "column": 48,
        "line": 18,
        "rule": "block-closing-brace-newline-before",
        "severity": "error",
        "text": "Expected newline before \\"}\\" (block-closing-brace-newline-before)",
      },
      Object {
        "column": 56,
        "line": 19,
        "rule": "block-closing-brace-newline-before",
        "severity": "error",
        "text": "Expected newline before \\"}\\" (block-closing-brace-newline-before)",
      },
      Object {
        "column": 41,
        "line": 20,
        "rule": "block-closing-brace-newline-before",
        "severity": "error",
        "text": "Expected newline before \\"}\\" (block-closing-brace-newline-before)",
      },
      Object {
        "column": 49,
        "line": 21,
        "rule": "block-closing-brace-newline-before",
        "severity": "error",
        "text": "Expected newline before \\"}\\" (block-closing-brace-newline-before)",
      },
      Object {
        "column": 31,
        "line": 22,
        "rule": "block-closing-brace-newline-before",
        "severity": "error",
        "text": "Expected newline before \\"}\\" (block-closing-brace-newline-before)",
      },
      Object {
        "column": 72,
        "line": 23,
        "rule": "block-closing-brace-newline-before",
        "severity": "error",
        "text": "Expected newline before \\"}\\" (block-closing-brace-newline-before)",
      },
      Object {
        "column": 61,
        "line": 24,
        "rule": "block-closing-brace-newline-before",
        "severity": "error",
        "text": "Expected newline before \\"}\\" (block-closing-brace-newline-before)",
      },
      Object {
        "column": 36,
        "line": 25,
        "rule": "block-closing-brace-newline-before",
        "severity": "error",
        "text": "Expected newline before \\"}\\" (block-closing-brace-newline-before)",
      },
      Object {
        "column": 42,
        "line": 26,
        "rule": "block-closing-brace-newline-before",
        "severity": "error",
        "text": "Expected newline before \\"}\\" (block-closing-brace-newline-before)",
      },
      Object {
        "column": 82,
        "line": 27,
        "rule": "block-closing-brace-newline-before",
        "severity": "error",
        "text": "Expected newline before \\"}\\" (block-closing-brace-newline-before)",
      },
      Object {
        "column": 48,
        "line": 28,
        "rule": "block-closing-brace-newline-before",
        "severity": "error",
        "text": "Expected newline before \\"}\\" (block-closing-brace-newline-before)",
      },
      Object {
        "column": 71,
        "line": 29,
        "rule": "block-closing-brace-newline-before",
        "severity": "error",
        "text": "Expected newline before \\"}\\" (block-closing-brace-newline-before)",
      },
      Object {
        "column": 39,
        "line": 30,
        "rule": "block-closing-brace-newline-before",
        "severity": "error",
        "text": "Expected newline before \\"}\\" (block-closing-brace-newline-before)",
      },
      Object {
        "column": 47,
        "line": 31,
        "rule": "block-closing-brace-newline-before",
        "severity": "error",
        "text": "Expected newline before \\"}\\" (block-closing-brace-newline-before)",
      },
      Object {
        "column": 4,
        "line": 13,
        "rule": "block-opening-brace-newline-after",
        "severity": "error",
        "text": "Expected newline after \\"{\\" (block-opening-brace-newline-after)",
      },
      Object {
        "column": 10,
        "line": 14,
        "rule": "block-opening-brace-newline-after",
        "severity": "error",
        "text": "Expected newline after \\"{\\" (block-opening-brace-newline-after)",
      },
      Object {
        "column": 10,
        "line": 15,
        "rule": "block-opening-brace-newline-after",
        "severity": "error",
        "text": "Expected newline after \\"{\\" (block-opening-brace-newline-after)",
      },
      Object {
        "column": 40,
        "line": 16,
        "rule": "block-opening-brace-newline-after",
        "severity": "error",
        "text": "Expected newline after \\"{\\" (block-opening-brace-newline-after)",
      },
      Object {
        "column": 41,
        "line": 17,
        "rule": "block-opening-brace-newline-after",
        "severity": "error",
        "text": "Expected newline after \\"{\\" (block-opening-brace-newline-after)",
      },
      Object {
        "column": 32,
        "line": 18,
        "rule": "block-opening-brace-newline-after",
        "severity": "error",
        "text": "Expected newline after \\"{\\" (block-opening-brace-newline-after)",
      },
      Object {
        "column": 33,
        "line": 19,
        "rule": "block-opening-brace-newline-after",
        "severity": "error",
        "text": "Expected newline after \\"{\\" (block-opening-brace-newline-after)",
      },
      Object {
        "column": 25,
        "line": 20,
        "rule": "block-opening-brace-newline-after",
        "severity": "error",
        "text": "Expected newline after \\"{\\" (block-opening-brace-newline-after)",
      },
      Object {
        "column": 26,
        "line": 21,
        "rule": "block-opening-brace-newline-after",
        "severity": "error",
        "text": "Expected newline after \\"{\\" (block-opening-brace-newline-after)",
      },
      Object {
        "column": 10,
        "line": 22,
        "rule": "block-opening-brace-newline-after",
        "severity": "error",
        "text": "Expected newline after \\"{\\" (block-opening-brace-newline-after)",
      },
      Object {
        "column": 22,
        "line": 23,
        "rule": "block-opening-brace-newline-after",
        "severity": "error",
        "text": "Expected newline after \\"{\\" (block-opening-brace-newline-after)",
      },
      Object {
        "column": 34,
        "line": 24,
        "rule": "block-opening-brace-newline-after",
        "severity": "error",
        "text": "Expected newline after \\"{\\" (block-opening-brace-newline-after)",
      },
      Object {
        "column": 20,
        "line": 25,
        "rule": "block-opening-brace-newline-after",
        "severity": "error",
        "text": "Expected newline after \\"{\\" (block-opening-brace-newline-after)",
      },
      Object {
        "column": 26,
        "line": 26,
        "rule": "block-opening-brace-newline-after",
        "severity": "error",
        "text": "Expected newline after \\"{\\" (block-opening-brace-newline-after)",
      },
      Object {
        "column": 61,
        "line": 27,
        "rule": "block-opening-brace-newline-after",
        "severity": "error",
        "text": "Expected newline after \\"{\\" (block-opening-brace-newline-after)",
      },
      Object {
        "column": 27,
        "line": 28,
        "rule": "block-opening-brace-newline-after",
        "severity": "error",
        "text": "Expected newline after \\"{\\" (block-opening-brace-newline-after)",
      },
      Object {
        "column": 55,
        "line": 29,
        "rule": "block-opening-brace-newline-after",
        "severity": "error",
        "text": "Expected newline after \\"{\\" (block-opening-brace-newline-after)",
      },
      Object {
        "column": 23,
        "line": 30,
        "rule": "block-opening-brace-newline-after",
        "severity": "error",
        "text": "Expected newline after \\"{\\" (block-opening-brace-newline-after)",
      },
      Object {
        "column": 24,
        "line": 31,
        "rule": "block-opening-brace-newline-after",
        "severity": "error",
        "text": "Expected newline after \\"{\\" (block-opening-brace-newline-after)",
      },
      Object {
        "column": 20,
        "line": 13,
        "rule": "declaration-block-semicolon-newline-after",
        "severity": "error",
        "text": "Expected newline after \\";\\" (declaration-block-semicolon-newline-after)",
      },
      Object {
        "column": 49,
        "line": 23,
        "rule": "declaration-block-semicolon-newline-after",
        "severity": "error",
        "text": "Expected newline after \\";\\" (declaration-block-semicolon-newline-after)",
      },
      Object {
        "column": 13,
        "line": 11,
        "rule": "function-url-quotes",
        "severity": "error",
        "text": "Unexpected quotes (function-url-quotes)",
      },
      Object {
        "column": 33,
        "line": 15,
        "rule": "function-url-quotes",
        "severity": "error",
        "text": "Unexpected quotes (function-url-quotes)",
      },
      Object {
        "column": 12,
        "line": 7,
        "rule": "no-eol-whitespace",
        "severity": "error",
        "text": "Unexpected whitespace at end of line (no-eol-whitespace)",
      },
      Object {
        "column": 48,
        "line": 31,
        "rule": "no-missing-end-of-source-newline",
        "severity": "error",
        "text": "Unexpected missing end-of-source newline (no-missing-end-of-source-newline)",
      },
      Object {
        "column": 1,
        "line": 14,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 1,
        "line": 15,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 1,
        "line": 16,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 1,
        "line": 17,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 1,
        "line": 18,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 1,
        "line": 19,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 1,
        "line": 20,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 1,
        "line": 21,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 1,
        "line": 22,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 1,
        "line": 23,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 1,
        "line": 24,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 1,
        "line": 25,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 1,
        "line": 26,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 1,
        "line": 27,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 1,
        "line": 28,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 1,
        "line": 29,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 1,
        "line": 30,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 1,
        "line": 31,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 17,
        "line": 27,
        "rule": "selector-list-comma-newline-after",
        "severity": "error",
        "text": "Expected newline after \\",\\" (selector-list-comma-newline-after)",
      },
      Object {
        "column": 38,
        "line": 27,
        "rule": "selector-list-comma-newline-after",
        "severity": "error",
        "text": "Expected newline after \\",\\" (selector-list-comma-newline-after)",
      },
      Object {
        "column": 26,
        "line": 29,
        "rule": "selector-list-comma-newline-after",
        "severity": "error",
        "text": "Expected newline after \\",\\" (selector-list-comma-newline-after)",
      },
      Object {
        "column": 13,
        "line": 11,
        "rule": "string-quotes",
        "severity": "error",
        "text": "Expected double quotes (string-quotes)",
      },
      Object {
        "column": 33,
        "line": 15,
        "rule": "string-quotes",
        "severity": "error",
        "text": "Expected double quotes (string-quotes)",
      },
    ],
  },
]
`;
